workers/unity/Assets/Playground/Scripts/Cubes/CollisionProcessSystem.cs

```diff
@@ -28,49 +34,51 @@ private struct Data
         {
             public readonly int Length;
             public EntityArray Entities;
-            public ComponentDataArray<SpatialOSLaunchable> Launchable;
+            [ReadOnly] public ComponentDataArray<SpatialOSLaunchable> Launchable; // Gets updated through PostUpdate
```

Comment:
Why are we doing that?

Reply:
Please put comments on the line above.

Reply:
@Ivaskovic You wrote this, why are we doing that?

Moved the comment

Reply:
Otherwise nothing got updated. Can't remember why, there used to be a comment that I deleted in one of the ancient commits.

Reply:
Discussed this on Friday. The reason is that every collision is registered on two entities, and we want updates to happen only after `OnUpdate()`. This should have a one or two line comment briefly explaining it.